src/toolbar/toolbar-title.jsx

```diff
@@ -1,23 +1,7 @@
 import React from 'react';
+import StylePropable from '../mixins/style-propable';
 import getMuiTheme from '../styles/getMuiTheme';
 
-function getStyles(props, state) {
-  const {
-    baseTheme,
-    toolbar,
-  } = state.muiTheme;
-
-  return {
```

Comment:
I think this PR accidentally undoes the migration for this component :grin:

@tintin1343 Could you see if you could reapply your changes to the latest version in master?